app/presenters/evidence_hub_search_form_presenter.rb

```diff
@@ -0,0 +1,15 @@
+class EvidenceHubSearchFormPresenter < BasePresenter
+  def translate_field(field)
+    object.class.human_attribute_name(field)
+  end
+
+  def filter_options(field)
+    translate_field(field).map do |option|
+      FilterOption.new(option)
+    end
+  end
+
+  def class_name
+    object.class.name.underscore
+  end
+end
```

Comment:
At the moment the form object is only a object with accessors without any methods of handling the form inputs. I really like to hear other's opinions on this, but IMHO this Presenter could be deleted and the filter options could be moved to the Form object, since sounds like Form object responsibility. By the way I didn't understand the class name method **purpose**. Is it important the html class name to be 'evidence_hub_search_form_presenter'?

Reply:
I was thinking that the form object will be responsible for validating the form input. e.g. check that the filter option value is valid, as in one of the values in the yml file. If it is not valid, should we reject it and carry on or return the user to the form. These values will be visible on the url.

The purpose of the `class_name` method prefaces the field parameters with the class name as opposed to class instance e.g.
```
<input class="checkbox" type="checkbox" value="Children (3 - 11)" name="evidence_hub_search_form[client_groups][]" id="evidence_hub_search_form_client_groups_children_3_-_11">
```
vs
```
<input class="checkbox" type="checkbox" value="Children (3 - 11)" name="#<EvidenceHubSearchForm:0x007ff77bcfb7f8>[client_groups][]" id="__EvidenceHubSearchForm:0x007ff77bcfb7f8_client_groups_children_3_-_11">
```